sdcm/tester.py

```diff
@@ -1194,12 +1201,12 @@ def clean_resources(self):
 
         if db_cluster_errors:
             self.log.error('Errors found on DB node logs:')
-            for node_errors in db_cluster_errors:
-                for node_name in node_errors:
-                    for (index, line) in node_errors[node_name]:
-                        self.log.error('%s: L%s -> %s',
-                                       node_name, index + 1, line.strip())
-            self.fail('Errors found on DB node logs (see test logs)')
+            for node_name, node_errors in db_cluster_errors.items():
+                for (index, line) in node_errors:
+                    self.log.error('%s: L%s -> %s',
+                                   node_name, index + 1, line.strip())
+            # TODO: think if we want to fail the test cause of this
+            # self.fail('Errors found on DB node logs (see test logs)')
```

Comment:
uncomment

Reply:
add TODO: comment to remove this logic after Analyzer will be implemented

Reply:
after looking again into this one, since we are adding a long list of failure to the list.
i.e. every stall can make the test fail. hence not much point anymore to fail base on the log events